data-mermaid/mermaid-dash
# [M482] Fix missing bleaching download

**Steps to test**:

- Find a project to look for bleaching in download modal.
- **Example**:
   -  Filter sites by project `2023 Coral Bleaching_CoralAtlas`.
   -  Select any site marker.
   -  Click download.
   -  Bleaching download should appear if it's in the project.
- **Screenshot**:
![Screenshot 2023-05-26 at 2 03 20 PM](https://github.com/data-mermaid/mermaid-dash/assets/6288799/79082b6f-7f01-4c0f-9405-b2dbe8ac7128)
